### DISCUSSION TITLE
运行转换检测模型模型时报错Illegal instruction

### DISCUSSION THREAD

**Discussion by KevinLLF** [PaddlePaddle/PaddleOCR#12220] [Q&A]
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/85452733/6af254bd-ca02-4622-873f-45bcf7e682da)
![image](https://github.com/PaddlePaddle/PaddleOCR/assets/85452733/583eb83f-8c44-411c-bc17-0c22faf6713b)
其他版本也试过 一样的问题

**Comment by visionwxc** [comment]
相同问题，请问解决了嘛 ？

**Comment by KevinLLF** [comment]
> 相同问题，请问解决了嘛 ？

因为机器不支持avx指令，换一个支持指令的cpu就可以了

**Comment by dlooto** [comment]
能否通过调整参数解决指令集的问题?
